reports/from-cypress.json

```diff
@@ -0,0 +1,40 @@
+{"/home/victory/Desktop/OSS/ambianic-ui/cypress/integration/ambianic-tests/about.spec.js": {"path":"/home/victory/Desktop/OSS/ambianic-ui/cypress/integration/ambianic-tests/about.spec.js","statementMap":{"0":{"start":{"line":2,"column":0},"end":{"line":33,"column":2}},"1":{"start":{"line":3,"column":2},"end":{"line":5,"column":4}},"2":{"start":{"line":4,"column":4},"end":{"line":4,"column":22}},"3":{"start":{"line":7,"column":2},"end":{"line":13,"column":4}},"4":{"start":{"line":8,"column":4},"end":{"line":9,"column":42}},"5":{"start":{"line":11,"column":4},"end":{"line":12,"column":57}},"6":{"start":{"line":15,"column":2},"end":{"line":20,"column":4}},"7":{"start":{"line":16,"column":14},"end":{"line":16,"column":37}},"8":{"start":{"line":17,"column":4},"end":{"line":17,"column":48}},"9":{"start":{"line":18,"column":4},"end":{"line":19,"column":54}},"10":{"start":{"line":22,"column":2},"end":{"line":27,"column":4}},"11":{"start":{"line":23,"column":14},"end":{"line":23,"column":37}},"12":{"start":{"line":24,"column":4},"end":{"line":24,"column":48}},"13":{"start":{"line":25,"column":4},"end":{"line":26,"column":49}},"14":{"start":{"line":29,"column":2},"end":{"line":32,"column":4}},"15":{"start":{"line":30,"column":4},"end":{"line":31,"column":48}}},"fnMap":{"0":{"name":"(anonymous_0)","decl":{"start":{"line":2,"column":21},"end":{"line":2,"column":22}},"loc":{"start":{"line":2,"column":27},"end":{"line":33,"column":1}},"line":2},"1":{"name":"(anonymous_1)","decl":{"start":{"line":3,"column":13},"end":{"line":3,"column":14}},"loc":{"start":{"line":3,"column":19},"end":{"line":5,"column":3}},"line":3},"2":{"name":"(anonymous_2)","decl":{"start":{"line":7,"column":33},"end":{"line":7,"column":34}},"loc":{"start":{"line":7,"column":39},"end":{"line":13,"column":3}},"line":7},"3":{"name":"(anonymous_3)","decl":{"start":{"line":15,"column":30},"end":{"line":15,"column":31}},"loc":{"start":{"line":15,"column":36},"end":{"line":20,"column":3}},"line":15},"4":{"name":"(anonymous_4)","decl":{"start":{"line":22,"column":30},"end":{"line":22,"column":31}},"loc":{"start":{"line":22,"column":36},"end":{"line":27,"column":3}},"line":22},"5":{"name":"(anonymous_5)","decl":{"start":{"line":29,"column":27},"end":{"line":29,"column":28}},"loc":{"start":{"line":29,"column":33},"end":{"line":32,"column":3}},"line":29}},"branchMap":{},"s":{"0":2,"1":2,"2":8,"3":2,"4":2,"5":2,"6":2,"7":2,"8":2,"9":2,"10":2,"11":2,"12":2,"13":2,"14":2,"15":2},"f":{"0":2,"1":8,"2":2,"3":2,"4":2,"5":2},"b":{},"_coverageSchema":"1a1c01bbd47fc00a2c39e90264f33305004495a9","hash":"d4de55937d062e0d9578189324a6e10f4d882989"}
```

Comment:
@vickywane what is the purpose of adding a coverage report to the github repo? Did you mean to add these files to .gitignore?